stdlib/codecs.pyi

```diff
@@ -239,7 +239,7 @@ class StreamReaderWriter(TextIO):
     def __next__(self) -> str: ...
     def __iter__(self: Self) -> Self: ...
     def write(self, data: str) -> None: ...  # type: ignore[override]
-    def writelines(self, list: Iterable[str]) -> None: ...
+    def writelines(self, list: Iterable[str]) -> None: ...  # type: ignore[override]  # https://github.com/python/mypy/issues/14002
```

Comment:
We probably don't need this `type: ignore` anymore (as well as several others), with the latest mypy release.

Reply:
Cleaned up all comments referencing that issue.